dapp/src/services/transaction-history.service.js

```diff
@@ -0,0 +1,15 @@
+export default class TransactionHistoryService {
+  constructor() {
+    this.baseURL = `${process.env.ANALYTICS_ENDPOINT}/api/v1/address`
+  }
+
+  async fetchHistory(account) {
+    const response = await fetch(
```

Comment:
Fyi, this doesn't handle errors correctly. See: https://react-query.tanstack.com/guides/query-functions#usage-with-fetch-and-other-clients-that-do-not-throw-by-default

Reply:
Thanks for spotting this - will send a fix.

Reply:
I missed that one as well. Hope you don't mind @evgenyboxer pushed a minor commit to fix it: https://github.com/OriginProtocol/origin-dollar/commit/6d42c191acc0d6e4ce97671dd8d0c6803c059b43

Reply:
looks good! thx.